test/serializers/cache_test.rb

```diff
@@ -3,6 +3,8 @@
 module ActiveModel
   class Serializer
     class CacheTest < Minitest::Test
+      include ActiveSupport::Testing::Stream
+
       def setup
         ActionController::Base.cache_store.clear
```

Comment:
Might as well get rid of this line, as it is now called before each test anyways.